examples/crm/src/deals/DealEdit.tsx

```diff
@@ -46,31 +50,17 @@ function EditHeader() {
     }
 
     return (
-        <DialogTitle
-            sx={{
-                paddingBottom: 0,
-            }}
-        >
-            <Stack
-                direction="row"
-                alignItems="center"
-                justifyContent="space-between"
-                spacing={1}
+        <DialogTitle>
+            Edit{' '}
+            <Typography
```

Comment:
**thought:** I'm not sure that this is a great UX to hide back to show in a link without any other information, but I do not have a better idea for this problem